static/js/apps/nl_interface/app.tsx

```diff
@@ -101,21 +102,26 @@ export function App(): JSX.Element {
   }
 
   useEffect(() => {
-    // If there are prompts in the url, automatically input the first prompt
-    // into the search box.
-    // If autoRun is enabled, runs every prompt (';' separated) from the url.
+    // If there are url prompts that have not been searched, input the next
+    // url prompt into the search box.
     // TODO: Do this by going through state/props instead of directly
     // manipulating the DOM.
-    if (urlPrompts.current.length) {
-      inputNextPrompt(false);
+    if (!urlPrompts.current.length) {
+      return;
+    }
```

Comment:
Before this change, when `urlPrompts.current.length` was 0, we were clearing on unmount but not anymore.  That is intentional?

Reply:
good catch, updated